dali/test/python/test_autoserialize.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvidia.dali._utils.autoserialize import invoke_autoserialize
+from nose.tools import raises
+
+serialized_filename = "/tmp/some_custom_name"
+
+
+def test_direct_import():
+    from .triton import decorated_function
+    invoke_autoserialize(decorated_function, serialized_filename)
+
+
+def test_indirect_import():
+    from .triton import imports_decorated_function
+    invoke_autoserialize(imports_decorated_function, serialized_filename)
+
+
+@raises(RuntimeError)
```

Comment:
How about checking for the error message if the expected error is triggered?

Reply:
Done